app/views/admin/contacts/_tab_heading.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="govuk-body">
```

Comment:
Do we need govuk-body here? I would've thought that the summary list component uses that styling by default?